f600233d1b154d507b782f84604bcb12/update/sys_script_include_b29002b097a211106fa8b4b3f153afc4.xml

```diff
@@ -9,35 +9,45 @@
         <name>Igce</name>
         <script><![CDATA[var Igce = Class.create();
 Igce.prototype = {
-
+    IGCE_ESTIMATE_TABLE: "x_g_dis_atat_igce_estimate",
+    IGCE_ESTIMATE_FIELDS: ["acquisition_package", "classification_instance", "classification_level", "contract_type", "cross_domain_solution",
+        "dow_task_number", "environment_instance", "idiq_clin_type", "title", "unit", "unit_price", "unit_quantity"],
+    ENV_INST_TABLE: "x_g_dis_atat_environment_instance",
+    CLASS_INST_TABLE: "x_g_dis_atat_classification_instance",
+    CROSS_DOMAIN_SOLN_TABLE: "x_g_dis_atat_cross_domain_solution",
+    SELECTED_PERIOD_FIELDS: ["need_for_entire_task_order_duration", "selected_periods"],
+    CLASSIFICATION_LEVEL_TABLE: "x_g_dis_atat_classification_level",
+    TRAVEL_REQ_TABLE: "x_g_dis_atat_travel_requirement",
+    TRAVEL_REQ_FIELDS: ["selected_periods", "duration_in_days", "number_of_travelers", "trip_location", "number_of_trips"],
+    REQ_COST_ESTIMATE_TABLE: "x_g_dis_atat_requirements_cost_estimate",
+    REQ_COST_ESTIMATE_FIELDS: ["travel_option", "travel_estimated_values", "surge_requirement_capabilities",// y/n choice
+        "surge_requirement_capacity",// int
+        "how_est_dev_contracting_office_other_charges_fee", // choice y/n
+        "how_est_dev_contracting_office_other_fee_percentage", // int
+        "how_est_dev_prev_cost_estimate_comp_option", //MORE_THAN / LESS_THAN / SAME
+        "how_est_dev_prev_cost_estimate_comp_percentage", // integer
+        "how_est_dev_cost_estimate_description", // string 1000 chars
+        "how_est_dev_tools_used", //String 100
+        "how_est_dev_other_tools_used", //string 100
+
+
+    ],
+    TRAINING_ESTIMATE_TABLE: "x_g_dis_atat_training_estimate",
+    TRAINING_ESTIMATE_FIELDS: ["acquisition_package",
+        "subscription_type", //choice
+        "training_estimated_values", //4000 string with periodsys_id: quantity
+        "training_option", //choice SINGLE is for all option periods MULTIPLE is for different
+        "training_unit", //PER_PERSON 	PER_CLASS ANNUAL_SUBSCRIPTION MONTHLY_SUBSCRIPTION
+        "estimated_price_per_training_unit", //price
+
+    ],
+    PERIOD_TABLE: "x_g_dis_atat_period",
+    PERIOD_FIELDS: ["period_type", "period_unit", "period_unit_count", "option_order"],
     acquisitionPackage: null,
 
     initialize: function (acquisitionPackage) {
         this.acquisitionPackage = acquisitionPackage;
     },
-
-    getEstimatedObligation: function (filterFn) {
```

Comment:
We need these two methods for CSS integration. Understanding that the structure has changed, is there anything blocking including them in the refactor?

Reply:
I saw this in the Igce script and didn't see actually used anywhere so I removed it, I will put those 2 methods back in but I think in the future we should try to include all functions in the scope of where they are actually being used to avoid confusion/overwriting.